src/main/java/uk/nhs/hee/trainee/details/mapper/CctMapper.java

```diff
@@ -37,6 +38,8 @@
 @Mapper(componentModel = SPRING)
 public interface CctMapper {
 
+  public static final LocalDate PLACEHOLDER_CCT_DATE = LocalDate.of(2030, 1, 1);
```

Comment:
LocalDate.MAX was too much for the FE to handle, so another placeholder date has been selected.